client/src/components/employer-form/index.js

```diff
@@ -202,6 +202,23 @@ export const Form = ({ hideCollectionNotice, initialValues, isDisabled }) => {
     }
   };
 
+  if (!initialValues && isDisabled) {
```

Comment:
This prevents setting initial values twice? ie:

1st render: `initialValues = undefined`
2nd render: `initialValues = {...initialValues}` // <-- wouldn't do anything because formik doesn't look for updates to initialValues?

If so, another low-weight solution could be the [enableReinitialize](https://formik.org/docs/api/formik#enablereinitialize-boolean) prop

Reply:
Neat! I had no idea that formik had already thought of this. I'll give it a try.